src/rules/spacing_after_comma.coffee

```diff
@@ -22,10 +22,28 @@ module.exports = class SpacingAfterComma
             @inRegex = false
             return
 
-        unless token.spaced or token.newLine or token.generated or
+        unless token.spaced or token.newLine or @isGenerated(token, tokenApi) or
                 @isRegexFlag(token, tokenApi)
+            return { token }
+
+    # Coffeescript does some code generation when using CSX syntax, and it adds
+    # brackets & commas that are not marked as generated. The only way to check
+    # these is to see if the comma has the same column number as the last token.
+    isGenerated: (token, tokenApi) ->
```

Comment:
This function may have to change if the underlying coffeescript lexer starts marking things as generated.